tests/core/job/test_job_manager.py

```diff
@@ -474,7 +476,7 @@ def _create_task(function, nb_outputs=1, name=None):
     output_dn_configs = [
         Config.configure_data_node(f"output{i}", "pickle", Scope.SCENARIO, default_data=0) for i in range(nb_outputs)
     ]
-    _DataManager._bulk_get_or_create({cfg for cfg in output_dn_configs})
+    _DataManager._bulk_get_or_create(dict(output_dn_configs))
```

Comment:
Actually, the `_bulk_get_or_create` takes a list as a parameter.

```suggestion
    _DataManager._bulk_get_or_create(output_dn_configs)
```